### ISSUE TITLE
[Bug] Claim status tool appeals endpoint timeout

### ISSUE THREAD

**Issue by annaswims** [department-of-veterans-affairs/va.gov-team#13547]
We're seeing 1-2k timeout errors a day in the appeals endpoint. 

prod- http://sentry.vfs.va.gov/vets-gov/platform-api-production/issues/154334/
staging - http://sentry.vfs.va.gov/vets-gov/platform-api-staging/issues/155848
Consider increasing the timeout (currently 15s) or getting appeals team to improve performance. 
https://dsva.slack.com/archives/C2ZBKPMND/p1600118651005100

[average latency](http://grafana.vfs.va.gov/d/000000050/backend-service-report?viewPanel=5&orgId=1) is  about 1.5

**Comment by annaswims** [comment]
Looks like latency was bad, then metrics were broken by a statsd change 😢

<img width="1526" alt="Screen Shot 2020-10-02 at 2 42 32 PM" src="https://user-images.githubusercontent.com/19188/94958460-90d2db80-04bd-11eb-8b8d-577cbb1ef9c0.png">

**Comment by annaswims** [comment]
[Ops is aware of metrics issue](https://dsva.slack.com/archives/C30LCU8S3/p1601652264017000).  It'll need to be fixed in order to validate this issue

**Comment by annaswims** [comment]
still happening. http://sentry10.vfs.va.gov/organizations/vsp/issues/115/events/?project=3

**Comment by annaswims** [comment]
we see ~650 timeouts a day now, but the timeout is now configurable.  Lighthouse makes more calls to appeals, so we shoudl talk to them if we want to change more.

**Comment by annaswims** [comment]
I did a tad more digging... What I learned is - Caseflow is super complicated and makes a ton of requests to their external services to get statuses and aggregates them for us. If any one of those requests raises a timeout error we'll get a timeout response, but what we're seeing is that caseflow is still aggregating responses by the time we hit our timeout. The good news is that caseflow caches the response for 20h so retrying the request in a bit (minute ?) will likely succeed.
